CHANGES.md

```diff
@@ -1,5 +1,20 @@
 # Changes
 
+## edge-21.7.4
+
+This release continues to focus on dependency updates. It also adds proxy a
+change that add a new `l5d-proxy-error` information header to distinguish
```

Comment:
nit: feels a little redundant to say it adds a change that added a further change

```suggestion
This release continues to focus on dependency updates. It also adds the
`l5d-proxy-error` information header to distinguish proxy generated errors
```